aggregator/src/test/java/com/redhat/cloud/notifications/models/EmailAggregation.java

```diff
@@ -26,9 +26,6 @@ public class EmailAggregation extends CreationTimestamped {
     @JsonProperty(access = READ_ONLY)
     private Integer id;
 
-    @Size(max = 50)
-    private String accountId;
```

Comment:
I suppose this is something we are deleting later - is there a list of the columns we are deleting?

Reply:
Yes, this is what I mentioned [here](https://github.com/RedHatInsights/notifications-backend/pull/1441#discussion_r959940772) (same DB field even though the Java class is different).

Reply:
This will be the only field removed for now.